src/utils/inputs.ts

```diff
@@ -35,5 +35,5 @@ export function getDelimitedArrayInput(name: string, options?: InputOptions): st
   const value = getInput(name, options);
   const sep = _.includes(value, '\n') ? '\n' : ',';
   // eslint-disable-next-line @typescript-eslint/unbound-method
-  return _.map(_.split(value, sep), _.trim);
+  return _.map(_.filter(_.split(value, sep)), _.trim);
```

Comment:
Could we instead run the filter after trimming? And add a test for inputs with spaces between their commas?

Reply:
Already have a test for that (see [this test](https://github.com/quizlet/commit-changes-action/blob/46a940e46b81b70f404e962712df13d1117d26b5/src%2Futils%2F__tests__%2Finputs.test.ts#L40)), but I'll reorder the operations 😄

Reply:
If there's already a test, how's it passing? Also `comman` -> `comma` in the test name 😁